### PR TITLE
73/70 search_autocomplete bobeshko

### PR DESCRIPTION
**Pull request by marrybbb**

### CODE REVIEW

File: controllers/wish-list-controller.js [hunk 1]
```diff
@@ -1,7 +1,82 @@
-const wishList = require('../models/wish-list.model')
+const wishList = require('../models/wish-list.model');
+const tokenservice = require('../services/token-service');
+const { jwtConf } = require('../config/config');
+const tokenService = new tokenservice(jwtConf);
+const { getSearchPlace } = require('../models/places.model');
+//const { getToken } = require('../utils/token')
```
Comment (AndriiDruzhytskyi) [inline controllers/wish-list-controller.js:6]: Please remove commented code.

File: controllers/wish-list-controller.js [hunk 2]
```diff
@@ -1,7 +1,82 @@
...
+        })
+}
+exports.wishListPost = async (req, res) => {
+    try {
+        const { comment, done, inProgress } = req.body;
+        const userJwt = req.header('authorization');
+        const tokenSplit = userJwt.split(" ");
+        const decodedJWT = await tokenService.verify(tokenSplit[1]);
+        // const token = getToken();
+        // console.log(token)
```
Comment (AndriiDruzhytskyi) [inline controllers/wish-list-controller.js]: Please remove commented code.

File: utils/token.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+function getToken() {
+const userJwt = req.header('authorization');
+const tokenSplit = userJwt.split(" ");
+ return decodedJWT = tokenService.verify(tokenSplit[1]);
+}
+exports.getToken;
```
Comment (AndriiDruzhytskyi) [inline utils/token.js:6]: Please remove this dead code.

**Review by andrii-kondratiuk — APPROVED** [review]

**Review by AndriiDruzhytskyi — APPROVED** [review]
Good work, ready to merge after small token processing improvements.

**Review by Dmitry528 — APPROVED** [review]

**Pull request merged**